root/static/scripts/release-editor/fields.js

```diff
@@ -309,6 +309,7 @@ class Medium {
         this.formatUnknownToUser = ko.observable(Boolean(data.id && !data.formatID));
 
         var tracks = data.tracks;
+        this.tracksUnknownToUser = ko.observable(Boolean(data.id && !tracks));
```

Comment:
Will `tracks` be null/undefined there, or an empty array? 'Cause in JS `![]` is false. (Since these files aren't Flow typed, maybe `data.id && !(tracks && tracks.length)` would be the safest.)